### ISSUE TITLE
How to trian my own data?

### ISSUE THREAD

**Issue by anna0709** [jinyu121/TensorFlow-ShuffleNet#2]
Hi, sorry to bother. I just want to know what is the dataset structure.what is in the `path/to/tiny-image-net/train`and `path/to/tiny-image-net/test` file? jpg or png images? and where is the groundtruth?

**Comment by jinyu121** [comment]
Hi~

I must inform you that this code is INCOMPLETE which with some bugs. I will fix them sooooooon.

I am using the [tiny-imagenet](https://tiny-imagenet.herokuapp.com/) and follow its file structure. The training folder goes like this:

```
train
    class 1
        some picture
        some picture
    class 2
        some picture
```

While the test folder goes like this:

```
test
    some picture
    some picture
```

and there is a text file recording the `file->class` mapping.

And all pictures can be any formats include `png` and `jpg`

**Comment by FB-wh** [comment]
hi, what is the content of "val_annotations.txt" ?
